src/registrar/models/domain.py

```diff
@@ -174,7 +174,8 @@ def available(cls, domain: str) -> bool:
         """Check if a domain is available."""
         if not cls.string_could_be_domain(domain):
             raise ValueError("Not a valid domain: %s" % str(domain))
-        req = commands.CheckDomain([domain])
+        domain_name = domain.lower()
```

Comment:
**Question:** To confirm, the logic is that if we preventative lowercase it in the registrar and then check it in EPP, correct?

Meaning that even if we try `Meoward` and `meoward` this will show availability as the same domain, and not two different domains?